database-plugins/src/main/java/io/cdap/plugin/db/common/DBBaseConfig.java

```diff
@@ -69,7 +69,7 @@
   private DBConnectorConfig connection;
 
   public Boolean getEnableAutoCommit() {
-    return enableAutoCommit;
+    return enableAutoCommit == null ? Boolean.TRUE : enableAutoCommit;
```

Comment:
let's add a constructor in this class to set the default settings for this incase this method is overriden in some classes

Reply:
it should be `false` here

Reply:
done :)

Reply:
making the method return boolean should avoid issues for overriding.